### PR TITLE
Rebase 86.0.4240.110

### PR DESCRIPTION
**Pull request by romandev**
Refs: #767

### CODE REVIEW

**Comment by romandev** [comment]
PTAL @jdragonbae @codeimpl

File: upstream/tb_chrome_android_java_res_xml_main_preferences.xml.patch [hunk 1]
```diff
@@ -1,28 +1,43 @@
 diff --git tb/chrome/android/java/res/xml/main_preferences.xml tb_diff/chrome/android/java/res/xml/main_preferences.xml
 --- tb/chrome/android/java/res/xml/main_preferences.xml
 +++ tb_diff/chrome/android/java/res/xml/main_preferences.xml
-@@ -7,20 +7,12 @@
+@@ -8,9 +8,11 @@
      android:orderingFromXml="false">
  
+     <org.chromium.chrome.browser.sync.settings.SyncPromoPreference
++        app:isPreferenceVisible="false"
```
Comment (jdragonbae) [inline upstream/tb_chrome_android_java_res_xml_main_preferences.xml.patch:8]: Looks good!

**Comment by jdragonbae** [comment]
LGTM !

**Comment by romandev** [comment]
@jdragonbae Thank you for review! But you missed "Approve"

**Review by jdragonbae — APPROVED** [review]
LGTM

**Pull request merged**